### PR TITLE
css pseudo classes | Adam Park 

### PR DESCRIPTION
**Pull request by Adam-SungMin-Park**
Screen Shot 2020-11-11 at 11 46 29

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.thead{ background-color:black;
+  color:white;
+}
+.tr1{
```
Comment (Zhiwutian) [inline css-pseudo-classes/styles.css:4]: While this works, you should be using the `nth-child()` pseudo selector which is mentioned in the instructions. Please refactor your CSS using `nth-child()`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Please update the code per the comment below.

**Review by Zhiwutian — APPROVED** [review]
👍

File: css-pseudo-classes/styles.css (hunk 1 above)
Comment (Adam-SungMin-Park) [reply css-pseudo-classes/styles.css]: noted.

**Pull request merged**